util/topgen/merge.py

```diff
@@ -213,6 +214,12 @@ def elaborate_instance(instance, block: IpBlock):
     if 'base_addr' in instance:
         del instance['base_addr']
 
+    # Default value if no value provided and otherwise convert string to bool
+    if 'generate_dif' not in instance:
+        instance['generate_dif'] = True
+    else:
+        instance['generate_dif'], _ = check_bool(instance['generate_dif'], False)
```

Comment:
The second value in check_bool's returned tuple indicates an error, so it should be checked.

Reply:
`check_bool` already prints an error message. I also checked the error code and raise a `ValueError` if necessary.

Reply:
Thanks